METRICS.md

```diff
@@ -0,0 +1,135 @@
+Usage-metrics-collector will expose the following metrics:
```

Comment:
In today's SIG Instrumentation triage meeting, we discussed possibly autogenerating this list, similar to the tooling recently added to k/k that generates the documentation for stable metrics.